src/ui/ContextMenu/items/MuteMenuItem.tsx

```diff
@@ -0,0 +1,64 @@
+import React, { ReactElement, ReactNode, useCallback, useState } from 'react';
+import { BaseChannel, User } from '@sendbird/chat';
+import { MenuItem } from '..';
+
+export interface MuteMenuItemProps {
+  channel: BaseChannel;
+  user: User & { isMuted: boolean };
+  className?: string | Array<string>;
+  children: ReactElement | ReactElement[] | ReactNode;
+  disable?: boolean;
+  dataSbId?: string;
+  onChange?: (channel: BaseChannel, user: User, isMuted: boolean) => void;
+  onError?: (reason: any) => void;
+}
+
+export const MuteMenuItem = ({
+  channel,
+  user,
+  className = '',
+  children,
+  disable = false,
+  dataSbId = '',
+  onChange = () => {},
+  onError = () => {},
+}: MuteMenuItemProps): ReactElement => {
+  const [isMuted, setIsMuted] = useState(user.isMuted);
+  const [isProcessing, setIsProcessing] = useState(false);
```

Comment:
The commonality between the `useState` and `useRef` is that the values are preserved even if the component re-renders.
The difference is that with `useState`, when the value changes, re-rendering occurs.
`useRef`, on the other hand, does not have any impact when the value changes.

If it's not related to the UI, using `useRef` is generally useful in typical situations.

Reply:
I just made a fix.